src/quo2/components/buttons/button/type_values.cljs

```diff
@@ -0,0 +1,186 @@
+(ns quo2.components.buttons.button.type-values
+  (:require [quo2.foundations.colors :as colors]
+            [quo2.theme :as quo2.theme]))
+
+(defn custom-color-type
+  [customization-color]
+  {:icon-color           colors/white
+   :icon-secondary-color colors/white-opa-70
+   :label-color          colors/white
+   :background-color     (colors/custom-color
```

Comment:
Unnecessary vertical space, we can write `(colors/custom-color customization-color 50)` on the same line.